dts/common/nordic/nrf9280.dtsi

```diff
@@ -74,6 +74,10 @@
 	reserved-memory {
 		#address-cells = <1>;
 		#size-cells = <1>;
+
+		suit_storage_partition: memory@e6b7000 {
+			reg = <0xe6b7000 DT_SIZE_K(40)>;
```

Comment:
According to the docs, currently you use 20kB (24kB with RFU of the same size as on nRF54H20).
Do you have something in mind for the remaining 16kB?

Reply:
Not ye.

I've considered the full remaining 20 kilobytes as RFU in this change
https://github.com/nrfconnect/sdk-secdom/pull/984/files#diff-5bba5d9834ddcbaa2463b81dd36e32b60c6f41b2985f3c60e03ca0155ee58470